docs/index.md

```diff
@@ -7,6 +7,35 @@ current focus lies on groove rolling in elongation grooves. The core packages co
 a first estimation of forces and torques occurring in a pass sequence. There is a flexible plugin system, able to modify
 and extend the model set available to describe the process.
 
+# Background and Purpose
+
+To the experience of the authors, the field of metal forming simulation is currently dominated by handcrafted, specialized and monolithic or commercial software tools.
+There is a large variety of modelling approaches for groove rolling processes published in scientific literature within the last century, but often not available as software implementation to the public.
+
+Groove rolling is a rather complex process, one has to deal with mechanical loads, material flow, heat flow and generation, microstructure evolution and others.
+The figure below gives an impression about the influencing factors and their interdependencies.
+
+![Influences on Groove Rolling Processes](img/chart_influence.svg)
+
+Some of them are not describable by physically based models, or only with unsuitable high effort in computational resources.
+For this reason, there are large amounts of empirical approaches describing the distinct aspects suitable for specific conditions and materials.
+These often give sufficient results at low computation cost, if respective material specific coefficients are available and specific conditions of validity are met.
+For some aspects, especially those regarding material behavior and evolution, specialized and commonly used software tools already exist.
+PyRolL can use them as a subroutine if an appropriate interface plugin is available.
+
+The PyRolL project as a whole aims to provide a flexible software basis and a growing collection of model packages for the distinct aspects of groove rolling.
+With this, a rolling engineer can:
+- exchange the used models
```

Comment:
This new text is really useful, thanks.

As a minor addition, would it be possible (maybe as additional bullet points in the list here) to state a typical prediction that can be made with the tool? From the current list, I can see how the tool can be broadly used, but what exactly it "predicts" is unclear to me. For example, maybe bullet points like "Estimate the final wire/rod/product geometry for a given set of rolling grove designs/profiles", "Estimate the rolling forces per rolling pass", "Estimate the mechanical properties of wire/rod/product after each rolling pass". These points are all obvious to the experienced user, but as a new user, I have to go through the details of the tutorials to understand what exactly it can do. Does that make sense?

Reply:
Yes, it does. I will add this.